Scripts/Utilities/UnityBrandColorScheme.cs

```diff
@@ -5,450 +5,474 @@
+    /// </summary>
+    public static class UnityBrandColorScheme
+    {
+        /// <summary>
+        /// Random used to aid in lack of repeating color choices
+        /// </summary>
+        static System.Random s_ColorRandom = new System.Random();
+
+        /// <summary>
+        /// Collection of Unity brand color swatches
+        /// </summary>
+        static readonly List<Color> s_ColorSwatches = new List<Color>();
+
+        /// <summary>
+        /// Cache standard color swatch collection size, to reduce further lookups
+        /// </summary>
+        static int s_ColorSwatchRange;
+        static int s_RandomSwatchColorPosition;
+        static int s_RandomGradientPairColorAPosition;
+
+        /// <summary>
+        /// Collection of curated Unity brand gradients; selected pairs of specific unity-brand color swatches
+        /// </summary>
+        static readonly List<GradientPair> s_CuratedGradientPairs = new List<GradientPair>();
+        static readonly List<GradientPair> s_CuratedLightGradientPairs = new List<GradientPair>();
+        static readonly List<GradientPair> s_CuratedDarkGradientPairs = new List<GradientPair>();
+
+        /// <summary>
+        /// Cache curated gradients collection size, to reduce further lookups
+        /// </summary>
+        static int s_RandomCuratedGradientPairPosition = -1; // Set to -1 in order to allow a zero entry to be initially set
+        static int s_RandomCuratedLightGradientPairPosition = -1;
+        static int s_RandomCuratedDarkGradientPairPosition = -1;
+
+        static GradientPair s_SessionGradient;
+        static GradientPair s_SaturatedSessionGradient;
+
+        static Color s_Red;
+        static Color s_RedLight;
+        static Color s_RedDark;
+
+        static Color s_Magenta;
+        static Color s_MagentaLight;
+        static Color s_MagentaDark;
+
+        static Color s_Purple;
+        static Color s_PurpleLight;
+        static Color s_PurpleDark;
+
+        static Color s_Blue;
+        static Color s_BlueLight;
+        static Color s_BlueDark;
+
+        static Color s_Cyan;
+        static Color s_CyanLight;
+        static Color s_CyanDark;
+
+        static Color s_Teal;
+        static Color s_TealLight;
+        static Color s_TealDark;
+
+        static Color s_Green;
+        static Color s_GreenLight;
+        static Color s_GreenDark;
+
+        static Color s_Lime;
+        static Color s_LimeLight;
+        static Color s_LimeDark;
+
+        static Color s_Yellow;
+        static Color s_YellowLight;
+        static Color s_YellowDark;
+
+        static Color s_Orange;
+        static Color s_OrangeLight;
+        static Color s_OrangeDark;
+
+        static Color s_DarkBlue;
+        static Color s_DarkBlueLight;
+
+        static Color s_Dark;
+        static Color s_Darker;
+        static Color s_Light;
+
+        // Unity Swatches 2016
+        public static Color red { get { return s_Red; } }
```

Comment:
Should we get rid of the extra new lines?

Reply:
yeah, you can shrink it down